package.json

```diff
@@ -27,14 +27,11 @@
         "onLanguage:cpp",
         "onLanguage:cuda-cpp",
         "onLanguage:objective-c",
-        "onLanguage:objective-cpp",
-        "onCommand:clangd.activate",
-        "onCommand:clangd.install",
-        "onCommand:clangd.update"
+        "onLanguage:objective-cpp"
     ],
     "main": "./out/bundle",
     "scripts": {
-        "esbuild": "esbuild ./src/extension.ts --bundle --outfile=out/bundle.js --external:vscode --format=cjs --platform=node",
+        "esbuild": "esbuild ./src/extension.ts --bundle --outfile=out/bundle.js --external:@aws-sdk/client-s3 --external:vscode --format=cjs --platform=node",
```

Comment:
why do we need aws-sdk dependency here?

Reply:
I believe this is saying that we *don't* bundle this package. I don't remember why I had to add this flag. Try removing it and see?

Reply:
sent out https://github.com/clangd/vscode-clangd/pull/815